### DISCUSSION TITLE
unaccent postgresql extension

### DISCUSSION THREAD

**Discussion by Ben212** [prisma/prisma#17530] [Q&A] [answered]
I try to use unaccent extension with prisma 4.9 it work with queryRaw but not with prisma.findMany() ?
I use nestjs/typescript code first my config is : 

  previewFeatures = ["clientExtensions","fullTextSearch", "postgresqlExtensions"]
  unaccent = true
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
  extensions = [unaccent]
}
 but I got error 'unaccent' does not exist in type 'StringFilter'
my query is 👍 return await this.prisma.carBody.findMany({
      where: {
        name: { unaccent: { contains: "élégant" } },
      },
    });

**Comment by nurul3101** [comment, marked as answer]
Hello @Ben212 👋 

The PostgreSQL extension preview feature allows you to represent PostgreSQL extensions in your Prisma schema. But that specific extension may add functionality that is not currently supported by Prisma, which is the case for the unaccent PostgreSQL extension.

The unaccent extension is not yet supported in the Prisma query language.
We have a related feature request here:
- https://github.com/prisma/prisma/issues/6428

If you can add a 👍 to the feature request, the it would help our product team in prioritising it.

For now, you would need to use the `$queryRaw` that you are doing.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
